tests/dibi/PdoDriver.providedConnection.phpt

```diff
@@ -10,35 +10,62 @@ use Tester\Assert;
 
 require __DIR__ . '/bootstrap.php';
 
+
+function buildPDOConnection(int $errorMode = NULL): PDO {
+	global $config;
+
+	// used to parse config, establish connection
+	$connection = new \Dibi\Connection($config);
+	$dibiDriver = $connection->getDriver();
+	\assert($dibiDriver instanceof \Dibi\Drivers\PdoDriver);
+
+	// hack: extract PDO connection from driver (no public interface for that)
+	$connectionProperty = (new ReflectionClass($dibiDriver))
+		->getProperty('connection');
+	$connectionProperty->setAccessible(TRUE);
+	$pdo = $connectionProperty->getValue($dibiDriver);
+	\assert($pdo instanceof PDO);
```

Comment:
I have tried to rethink a little creation process of PDO connection. As it is quite complex I have delegated it back to `\Dibi\Connection`.